core/src/main/scala/algebra/lattice/DeMorgan.scala

```diff
@@ -0,0 +1,55 @@
+package algebra
+package lattice
+
+import scala.{specialized => sp}
+
+/**
+  * De Morgan algebras are bounded lattices that are also equipped with
+  * an involutive `complement` operation.
```

Comment:
If it is not necessarily a _complement_, let's not call it that. Just call it _De Morgan involution_ instead?

Reply:
Yes, De Morgan involution is a better name. I will change it :+1: